lib/cli.js

```diff
@@ -400,37 +372,29 @@ const cli = {
 			return 2;
 		}
 
-		const ActiveESLint = usingFlatConfig ? ESLint : LegacyESLint;
-
-		/** @type {ESLint|LegacyESLint} */
+		/** @type {ESLint} */
 		let engine;
 
-		if (options.concurrency && options.concurrency !== "off") {
+		if (options.concurrency !== "off") {
```

Comment:
`options.concurrency` is always a truthy value now. Previously, it was unset in eslintrc mode.